src/batch/src/rpc/service/exchange.rs

```diff
@@ -13,23 +13,47 @@
 // limitations under the License.
 
 use risingwave_common::error::{Result, ToRwResult};
-use risingwave_pb::task_service::GetDataResponse;
+use risingwave_pb::common::Status as PbStatus;
+use risingwave_pb::data::DataChunk;
+use risingwave_pb::task_service::{ExecuteResponse, GetDataResponse};
 use tonic::Status;
 
-type ExchangeDataSender = tokio::sync::mpsc::Sender<std::result::Result<GetDataResponse, Status>>;
+type ExchangeDataSender<T> = tokio::sync::mpsc::Sender<std::result::Result<T, Status>>;
+
+pub trait CreateDataResponse: Send {
```

Comment:
Have to add this trait to unify both types of `Response`...

Reply:
Is there any other way of solving this? @liurenjie1024 
I don't know whether we will add another Protobuf type as `Response` for some other RPC in the future. 
May `impl` them by macro later if there are many.

Reply:
Sorry for the late reply. The `GetDataResponse` and `ExecuteReponse` are same, so I think using one proto struct should be enough? And we don't need this trait.

Reply:
`buf lint` disallows using the same type as the response type of two different RPC 😓

Reply:
Or we may be able to configure it in allowing this, let me check it.